src/python/turicreate/visualization/_plot.py

```diff
@@ -360,6 +360,8 @@ def display_table_in_notebook(sf, title=None):
     from PIL import Image
 
     import base64
+    import cgi
```

Comment:
Please put these imports within the if/else conditions where they are used. Otherwise one of the imports may fail on Python 2 or Python 3 (I think the html module is new in Python 3).

Reply:
@znation Thanks for your suggestion! Changes are implemented and request for PR